src/pages/Login/Login.js

```diff
@@ -1,8 +1,131 @@
 import React, { Component } from 'react';
+import IdPwInput from './IdPwInput';
+import './Login.scss';
 
 class Login extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      id: '',
+      pw: '',
+      btn: false,
+    };
+  }
+
+  handleInputs = e => {
+    const { name, value } = e.target;
+    this.setState({
+      [name]: value,
+    });
+  };
+
+  handleLogin = () => {
+    console.log('click');
+    fetch('http://10.58.4.133:8000/user/signin', {
+      method: 'POST',
+      body: JSON.stringify({
+        email: this.state.id,
+        password: this.state.pw,
+      }),
+    })
+      .then(result => result.json())
+      .then(result => {
+        if (result.MESSAGE === 'SUCCESS') {
+          alert(`Welcome ${this.state.id}`);
+          localStorage.setItem('dot-token', result.token);
+          this.props.history.push('/');
+        } else {
+          alert('Login failed');
+        }
+      });
+  };
+
   render() {
-    return <div></div>;
+    const IDPW = [
+      {
+        name: 'id',
+        title: '아이디',
+        type: 'text',
+        onchanging: this.handleInputs,
+      },
+      {
+        name: 'pw',
+        title: '비밀번호',
+        type: 'password',
+        onchanging: this.handleInputs,
+      },
+    ];
+
+    const checkIdPw =
+      this.state.id.includes('@') && this.state.pw.length > 7 ? true : false;
+    console.log(this.state);
```

Comment:
```suggestion
    const checkIdPw =
      this.state.id.includes('@') && this.state.pw.length > 7;
```
삼항연산자를 사용하지 않아도 해당 조건 자체가 true, false 값으로 매 렌더시마다 값을 가집니다!